.gitignore

```diff
@@ -61,3 +61,4 @@ examples/public/
 documentation/.docusaurus
 documentation/generated
 documentation/docs/api
+/react-components/stories/public/cadModel

```

Comment:
Can we add e.g. a `public/local` folder where we can put custom models like this? Just so that we don't have to update the ignore file every time we start working with a new custom model

Reply:
(Also, the leading `/` shouldn't be necessary)